src/events/channelCreate.ts

```diff
@@ -0,0 +1,17 @@
+import { Client as DiscordClient, GuildChannel } from "discord.js";
+import ServerSettingsRepository from "../repository/serverSettings";
+import { SetMutedPermissionsForChannel } from "../lib/mutedRole";
+
+export default async function ChannelCreateEvent(discordClient: DiscordClient, channel: GuildChannel) {
+	let serverSettings = await ServerSettingsRepository.GetByGuildId(channel.guild.id)
+	if (serverSettings === null) {
+		return;
+	}
+
+	if (serverSettings!.muteRole !== null) {
```

Comment:
Remove the `serverSettings === null` check and place `serverSettings?.muteRole != null` here